src/txmempool.h

```diff
@@ -579,6 +579,8 @@ class CTxMemPool
      */
     std::map<uint256, uint256> m_unbroadcast_txids GUARDED_BY(cs);
 
+    void ClearPrioritisation(const uint256 hash) EXCLUSIVE_LOCKS_REQUIRED(cs);
```

Comment:
3c7d4cb52869d5943c2f36b3b8a91bb60344274c

nit, could refactor to `const uint256&?`